src/ucs/datastruct/ptr_array.h

```diff
@@ -99,19 +122,22 @@ void ucs_ptr_array_remove(ucs_ptr_array_t *ptr_array, unsigned index,
 
 /**
  * Replace pointer in the array
- * @param  index    index of slot
- * @param  new_val  value to put into slot given by index
- * @return old value of the slot
+ *
+ * @param [in] ptr_array Pointer to a ptr array.
+ * @param [in] index     index of slot
+ * @param [in] new_val   value to put into slot given by index
+ * @return               old value of the slot
```

Comment:
pls fix the `@return` section as suggested below

Reply:
?
I saw the return needs to be separated by a space line in string.h coding conversion - So I added a new line before each @return.

Reply:
> ?
> I saw the return needs to be separated by a space line in string.h coding conversion - So I added a new line before each @return.

right! thank you